pyvelox/pyvelox.h

```diff
@@ -416,6 +417,25 @@ inline void addVectorBindings(
   m.def("from_list", [](const py::list& list) mutable {
     return pyListToVector(list, PyVeloxContext::getInstance().pool());
   });
+
+  m.def("export_to_arrow", [](VectorPtr& inputVector) {
+    auto arrowArray = new ArrowArray();
+    std::shared_ptr<facebook::velox::memory::MemoryPool> pool_{
+        facebook::velox::memory::getDefaultMemoryPool()};
+    facebook::velox::exportToArrow(inputVector, *arrowArray, pool_.get());
+    inputVector.reset();
+    return reinterpret_cast<uintptr_t>(arrowArray);
+  });
+
+  m.def(
+      "import_from_arrow",
+      [](uintptr_t arrowArrayPtr, uintptr_t arrowSchemaPtr) {
```

Comment:
Going back to previous question if I was to do  
` a = import_from_arrow(export_to_arrow(inputVector), pa.int64())` would there be a memory leak?

Reply:
Should work with the smart pointers. Added a roundtrip test for this case.